### ISSUE TITLE
Support Chinese

### ISSUE THREAD

**Issue by UchihaKutter** [MetaMask/metamask-extension#3077]
There is a huge demand for MetaMask in China. It's unconvenient to use it because of language. Please support chinese.

**Comment by danfinlay** [comment]
This is currently blocked by #437, which we have a bounty on, and someone working on. That will allow us to add translations.

**Comment by kumavis** [comment]
Closing in favor of https://github.com/MetaMask/metamask-extension/issues/3529 ( and https://github.com/MetaMask/metamask-extension/issues/3540 )
